recipes-bsp/firmware-pack/bootparameter-native_1.0.bb

```diff
@@ -0,0 +1,23 @@
+SECTION = "bootloaders"
+DESCRIPTION = "Application to create binaries in the correct format for rzg2l board flashing"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0-only;md5=801f80980d171dd6425610833a22dbe6"
+
+inherit native
+
+# Set S variable point to source file bootparameter.c
+S = "${WORKDIR}/sources"
+UNPACKDIR = "${S}"
```

Comment:
You don't seem to use the UNPACKDIR variable. Is it redundant?

Reply:
UNPACKDIR which is where the fetcher is asked to unpack fetched sources .In this situation, source file bootparameter.c is required and need to change UNPACKDIR to the correct directory . If UNPACKDIR is not correct ,this can lead to issues like missing source file.
Reference link: https://patchwork.yoctoproject.org/project/oe-core/patch/20240501215643.3386307-1-richard.purdie@linuxfoundation.org/#19085